pkg/events/events.go

```diff
@@ -1,57 +1,25 @@
 package events
 
 import (
-	"fmt"
-	"time"
-
-	"github.com/filecoin-project/go-state-types/big"
 	"github.com/filecoin-project/lassie/pkg/types"
-	"github.com/ipfs/go-cid"
 	"github.com/libp2p/go-libp2p/core/peer"
 	"github.com/multiformats/go-multicodec"
 )
 
-var (
```

Comment:
I'm not really a fan of the file sprawl in here, they're all such small files, making it harder to edit things in bulk (e.g. can't use smart editor techniques to make changes to all of them) and harder to see commonality across them

Reply:
So I debated this to myself while making this. I personally didn't like how unweildy the previous single file became. I enjoy this more because the events themselves should be small, individual, and unrelated to other events, and I think this portrays that well.